### ISSUE TITLE
AO entities "name" is missing for individuals

### ISSUE THREAD

**Issue by patphongs** [fecgov/openFEC#4744]
### Summary
The AO entities "name" field is missing for individuals. We suspect this may be a problem for AOs 2019-14 through 2021-02.

Example: https://stage.fec.gov/data/legal/advisory-opinions/2020-04/
There's a missing name for this individual in the AO entities table.

API call: https://api.open.fec.gov/v1/legal/docs/advisory_opinions/2020-04?api_key=DEMO_KEY

![Screen Shot 2021-01-21 at 9 47 04 AM](https://user-images.githubusercontent.com/12799132/105366712-a6375700-5bcd-11eb-8579-c42381d9c23f.png)
![Screen Shot 2021-01-21 at 9 46 51 AM](https://user-images.githubusercontent.com/12799132/105366713-a6375700-5bcd-11eb-9309-ebddfbcb10bc.png)

### Related issues
#4443 

**Here is the query to identify missing entities:**
```
 SELECT 
     entity.name as entity_name,
     ao_parsed.ao_id, 
     ao_parsed.ao_no, 
     ao_parsed.name, 
     ao_parsed.summary, 
     ao_parsed.req_date, 
     ao_parsed.issue_date, 
     ao.stage 
     
    FROM aouser.aos_with_parsed_numbers ao_parsed 
    INNER JOIN aouser.ao ao 
     ON ao_parsed.ao_id = ao.ao_id 
    INNER JOIN aouser.players p ON p.ao_id = ao.ao_id
    INNER JOIN aouser.entity entity USING (entity_id)
    INNER JOIN aouser.entity_type et ON et.entity_type_id = entity.type
    INNER JOIN aouser.role r USING (role_id)
	WHERE entity.name=''
    ORDER BY ao_parsed.ao_year, ao_parsed.ao_serial 
	
```

**Comment by pkfec** [comment]
| ao_no  |
|---------|
1990-23
1992-33
1993-17
1995-25
1998-18
1999-05
1999-25
1999-36
2000-09
2000-12
2000-23
2000-24
2001-11
2001-16
2001-17
2002-13
2002-13
2002-14
2002-14
2003-03
2003-10
2003-11
2003-12
2003-23
2003-23
2003-25
2003-37
2003-37
2003-37
2004-03
2004-05
2004-05
2004-12
2004-30
2004-30
2004-30
2004-30
2004-31
2004-31
2004-31
2004-35
2004-35
2004-35
2004-37
2004-38
2004-38
2004-38
2004-38
2004-39
2004-39
2004-39
2004-39
2004-43
2004-43
2004-43
2004-43
2004-43
2004-45
2004-45
2004-45
2004-45
2005-10
2005-10
2005-12
2005-13
2005-13
2005-13
2005-13
2005-13
2005-16
2005-16
2005-16
2006-10
2006-10
2006-10
2006-10
2006-11
2006-11
2006-11
2006-11
2006-13
2006-14
2006-14
2006-14
2006-14
2006-19
2006-19
2006-19
2006-19
2006-20
2006-20
2006-20
2006-20
2006-24
2006-24
2006-24
2006-24
2006-30
2006-30
2006-31
2006-31
2006-31
2006-31
2006-32
2006-32
2006-32
2006-32
2006-34
2007-03
2007-03
2007-03
2007-03
2007-04
2007-04
2007-04
2007-07
2007-09
2007-09
2007-09
2007-09
2007-11
2007-11
2007-11
2007-11
2007-11
2007-27
2007-27
2007-28
2007-28
2007-28
2007-28
2007-28
2007-32
2007-32
2007-32
2007-32
2007-33
2007-33
2007-33
2007-33
2007-35
2008-10
2008-14
2008-15
2008-15
2009-04
2009-04
2009-04
2009-13
2009-13
2009-13
2010-03
2010-03
2010-03
2010-07
2010-07
2010-07
2010-08
2010-08
2010-08
2010-08
2010-09
2010-11
2010-20
2010-20
2010-20
2010-24
2010-28
2011-09
2011-09
2011-09
2011-09
2011-11
2011-11
2011-11
2011-11
2011-11
2011-11
2011-11
2011-11
2011-11
2011-11
2011-11
2011-11
2011-11
2011-11
2011-12
2011-12
2011-12
2011-12
2011-14
2011-14
2011-21
2011-21
2011-21
2011-21
2011-21
2011-23
2011-23
2011-23
2011-23
2011-28
2011-28
2012-01
2012-11
2012-11
2012-11
2012-11
2012-14
2012-14
2012-14
2012-14
2012-14
2012-14
2012-17
2012-17
2012-19
2012-19
2012-19
2012-19
2012-26
2012-30
2012-31
2012-32
2012-33
2013-02
2013-02
2013-09
2013-09
2013-09
2013-09
2013-09
2013-10
2013-10
2013-10
2013-10
2013-13
2013-15
2013-16
2013-17
2013-18
2014-10
2014-16
2015-09
2015-09
2015-09
2015-09
2015-09
2015-16
2016-16
2016-21
2017-05
2017-12
2018-07
2018-12
2019-14
2019-15
2019-16
2019-18
2020-01
2020-01
2020-02
2020-03
2020-03
2020-03
2020-04
2020-06
2021-01
2021-02

**Comment by pkfec** [comment]
This issue is fixed in PR #4760. After PR #4760 gets merged and deployed to `Dev` space,  have to reload  task to load the AO's  data into elasticsearch service. I will create a follow up issue to track the AO data reload process across all 3 environments.
